hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rolloutgroup/RolloutGroupBeanQuery.java

```diff
@@ -151,10 +158,17 @@ protected void saveBeans(final List<ProxyRolloutGroup> arg0, final List<ProxyRol
     @Override
     public int size() {
         long size = 0;
-        if (null != rolloutId) {
-            firstPageRolloutGroupSets = getRolloutGroupManagement().findAllRolloutGroupsWithDetailedStatus(rolloutId,
-                    new PageRequest(0, SPUIDefinitions.PAGE_SIZE, sort));
-            size = firstPageRolloutGroupSets.getTotalElements();
+        if (rolloutId != null) {
```

Comment:
When is the rolloutId `null`? 
I guess if the rollout was deleted. Then i would prefer private method `boolean isRolloutDeleted`